<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ubuntu" name="com.rc.wefunit.probes.AssertTest" tests="9" failures="0" timestamp="20 May 2015 16:35:58 GMT" time="0.016" errors="0">
  <testcase name="method_assertTrue_throw_java_lang_AssertionError_when_argument_is_false" time="0.000" classname="com.rc.wefunit.probes.AssertTest"/>
  <testcase name="class_Assert_existence" time="0.001" classname="com.rc.wefunit.probes.AssertTest"/>
  <testcase name="method_assertTrue_no_Throwable_when_argument_is_true" time="0.001" classname="com.rc.wefunit.probes.AssertTest"/>
  <testcase name="method_assertNotNull_with_one_argument_Object" time="0.007" classname="com.rc.wefunit.probes.AssertTest"/>
  <testcase name="method_fail_with_no_params" time="0.000" classname="com.rc.wefunit.probes.AssertTest"/>
  <testcase name="method_assertNotNull_throw_AssertionError_no_Throwable_when_object_is_not_null" time="0.000" classname="com.rc.wefunit.probes.AssertTest"/>
  <testcase name="method_fail_with_string_argument" time="0.000" classname="com.rc.wefunit.probes.AssertTest"/>
  <testcase name="method_assertTrue_is_public_static_with_only_one_argument_Object" time="0.000" classname="com.rc.wefunit.probes.AssertTest"/>
  <testcase name="method_assertNotNull_throw_AssertionError_when_object_is_null" time="0.007" classname="com.rc.wefunit.probes.AssertTest"/>
</testsuite> <!-- com.rc.wefunit.probes.AssertTest -->
